### PR TITLE
[4.0][CLI] check install

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #31899 .

### Summary of Changes
check for install 1st

### Testing Instructions

1. Download Joomla 4 
2. DONT INSTALL JOOMLA
3. run `php cli/joomla.php`

### Actual result BEFORE applying this Pull Request
error

### Expected result AFTER applying this Pull Request
a message to install Joomla before using cli

### CODE REVIEW

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 69c03511d7589e40fa449a2734c2fb96c5a0579a<br /><br />Tested successfully in Beta7-dev of 9 January.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31938">issues.joomla.org/tracker/joomla-cms/31938</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 69c03511d7589e40fa449a2734c2fb96c5a0579a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31938">issues.joomla.org/tracker/joomla-cms/31938</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31938">issues.joomla.org/tracker/joomla-cms/31938</a>.</sub>

File: cli/joomla.php [hunk 1]
```diff
@@ -45,6 +45,14 @@
 	exit;
 }
 
+// Check if installed
+if (!file_exists(JPATH_ROOT . '/configuration.php'))
```
Comment (PhilETaylor) [inline cli/joomla.php]: Should this not be using JPATH_CONFIGURATION? as the configuration.php location could be anywhere!

Should it also not take into account if the /installation folder is still present?

eg smiting akin to https://github.com/joomla/joomla-cms/blob/a80d9d9e63d44faea6ee1d57185769beea420b26/includes/framework.php#L28

Comment (alikon) [reply cli/joomla.php]: it can be done better 
I agree

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on f98ecb65314b96a5b325f621cbffc660bf3b31f4<br /><br />Tested successfully in Beta7-dev of 10 January.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31938">issues.joomla.org/tracker/joomla-cms/31938</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on f98ecb65314b96a5b325f621cbffc660bf3b31f4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31938">issues.joomla.org/tracker/joomla-cms/31938</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31938">issues.joomla.org/tracker/joomla-cms/31938</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**